birthdays-project/BirthdayApp/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+
+class Birthdate(models.Model):
+    dob = models.DateField(max_length=8)
+
+    # Displays date in a dd-MMM-yy format.
+    def __str__(self):
+        return '{:%d %b %y}'.format(self.dob)
+
+
+# Creates a user Model.
+class User(models.Model):
```

Comment:
I challenge the name. To me, it's rather a `Person` than a `User` in the context of this app. Plus, django already has a `User` table. The names won't clash but it might confuse other developers and it will create problems if you have to import both models:

```
from django.contrib.auth.models import User
from .models import User as BirthdayUser  # importing with 'as' as one solution to resolve the conflict
```